pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java

```diff
@@ -962,10 +966,15 @@ public void accept(Notification n) {
                 log.debug("Received updated load report from broker node - [{}], scheduling re-ranking of brokers.",
                         n.getPath());
             }
-            scheduler.submit(this::updateRanking);
+            updateRanking = scheduler.submit(this::updateRanking);
         }
     }
 
+    @VisibleForTesting
+    Future<?> getUpdateRanking(){
```

Comment:
you make make this "public" , it is an "Impl" class, it is not public API

Reply:
Ok